Data/ByteString/FastBuilder/Internal.hs

```diff
@@ -462,19 +467,26 @@ toStrictByteString builder = unsafePerformIO $ do
 
 -- | Output a 'Builder' to a 'IO.Handle'.
 hPutBuilder :: IO.Handle -> Builder -> IO ()
-hPutBuilder !h builder = hPutBuilderWith h 100 4096 builder
+hPutBuilder !h builder = void $ hPutBuilderLen h builder
+{-# INLINE hPutBuilder #-}
+
+-- | Output a 'Builder' to a 'IO.Handle'. Returns the number of bytes written.
+hPutBuilderLen :: IO.Handle -> Builder -> IO Int
+hPutBuilderLen !h builder = hPutBuilderWith h 100 4096 builder
```

Comment:
Oops I forgot to add INLINE here.

Reply:
I don't think it's a problem. The old version of `hPutBuilder` didn't have an INLINE pragma either.